crates/astria-bridge-withdrawer/src/metrics.rs

```diff
@@ -26,6 +29,27 @@ impl Metrics {
         describe_gauge!(CURRENT_NONCE, Unit::Count, "The current nonce");
         let current_nonce = gauge!(CURRENT_NONCE);
 
+        describe_counter!(
+            NONCE_FETCH_COUNT,
+            Unit::Count,
+            "The number of times a fetch nonce request to sequencer was made"
+        );
+        let nonce_fetch_count = counter!(NONCE_FETCH_COUNT);
+
+        describe_counter!(
+            NONCE_FETCH_FAILURE_COUNT,
+            Unit::Count,
+            "The number of failed attempts to fetch nonce from sequencer"
+        );
+        let nonce_fetch_failure_count = counter!(NONCE_FETCH_FAILURE_COUNT);
+
+        describe_histogram!(
+            NONCE_FETCH_LATENCY,
```

Comment:
I think the metric name should include the units, but we can probably leave as is for now and review all the metrics' names in a later PR.